### ISSUE TITLE
`new_relic_phoenix` is deprecated; is this package being deprecated too for similar reasons?

### ISSUE THREAD

**Issue by doughsay** [binaryseed/new_relic_absinthe#18]
Just a question, as the title suggests.  Just want to stay up to date on things and want to know how things will be going.

**Comment by binaryseed** [comment]
Yes, the same thing will happen, but I have no timeline at the moment

**Comment by eprothro** [comment]
Is there any update on this regarding direction or timing? We're attempting to switch from AppSignal to New Relic for error reporting. 

This integration is working partially, so we're wondering if we should dig in here to improve this integration, or is there another effort underway to pull this into the `new_relic_agent`?
